docs/book/src/tasks/upgrading-clusters.md

```diff
@@ -47,6 +47,28 @@ that if a specific machine image is specified, it has to match the Kubernetes ve
 `KubeadmControlPlane` spec. In order to only trigger a single upgrade, the new `MachineTemplate` should be created first
 and then both the `Version` and `InfrastructureTemplate` should be modified in a single transaction.
 
+#### How to schedule a machine rollout
```

Comment:
```suggestion
#### How to schedule the next machine rollout
```

Reply:
So, to me this wording would imply that this gives me control over when the **next** rollout will happen, and as I mentioned I wanted to be explicit that this is not the case as the next rollout could happen before the time set in `UpgradeAfter`, e.g.:

- 08:20 I set `UpgradeAfter` to `2021-03-16T12:00:00Z`
- 10:00 I change Kubernetes version in `KubeadmControlPlane.spec`, the next rollout is triggered immediately
- 12:00 the scheduled rollout gets triggered

WDYT?

Reply:
That makes sense (wording is hard!)